extensions/cornerstone-dicom-pmap/src/getSopClassHandlerModule.ts

```diff
@@ -0,0 +1,224 @@
+  displaySet.referencedSeriesInstanceUID = referencedSeries.SeriesInstanceUID;
+
+  displaySet.getReferenceDisplaySet = () => {
+    const { displaySetService } = servicesManager.services;
+
+    if (displaySet.referencedDisplaySetInstanceUID) {
+      return displaySetService.getDisplaySetByUID(displaySet.referencedDisplaySetInstanceUID);
+    }
+
+    const referencedDisplaySets = displaySetService.getDisplaySetsForSeries(
+      displaySet.referencedSeriesInstanceUID
+    );
+
+    if (!referencedDisplaySets || referencedDisplaySets.length === 0) {
+      throw new Error('Referenced displaySet is missing for the parametric map');
+    }
+
+    const referencedDisplaySet = referencedDisplaySets[0];
+
+    displaySet.referencedDisplaySetInstanceUID = referencedDisplaySet.displaySetInstanceUID;
+
+    return referencedDisplaySet;
+  };
+
+  displaySet.getReferencedVolumeId = () => {
+    if (displaySet.referencedVolumeId) {
+      return displaySet.referencedVolumeId;
+    }
+
+    const referencedDisplaySet = displaySet.getReferenceDisplaySet();
+    const referencedVolumeURI = referencedDisplaySet.displaySetInstanceUID;
+    const referencedVolumeId = `${VOLUME_LOADER_SCHEME}:${referencedVolumeURI}`;
+
+    displaySet.referencedVolumeURI = referencedVolumeURI;
+    displaySet.referencedVolumeId = referencedVolumeId;
+
+    return referencedVolumeId;
+  };
+
+  displaySet.load = async ({ headers }) =>
+    await _load(displaySet, servicesManager, extensionManager, headers);
+
+  return [displaySet];
+}
+
+async function _load(
+  displaySet,
+  servicesManager: AppTypes.ServicesManager,
+  extensionManager,
+  headers
+) {
+  const volumeId = `${VOLUME_LOADER_SCHEME}:${displaySet.displaySetInstanceUID}`;
+  const volumeLoadObject = cache.getVolumeLoadObject(volumeId);
+
+  if (volumeLoadObject) {
+    return volumeLoadObject.promise;
+  }
+
+  displaySet.loadStatus.loading = true;
+
+  // We don't want to fire multiple loads, so we'll wait for the first to finish
+  // and also return the same promise to any other callers.
+  // loadPromises[SOPInstanceUID] = new Promise(async (resolve, reject) => {
+  const promise = _loadParametricMap({
+    extensionManager,
+    servicesManager,
+    displaySet,
+    headers,
+  });
+
+  cache.putVolumeLoadObject(volumeId, { promise }).catch(err => {
+    throw err;
+  });
+
+  promise
+    .then(() => {
+      displaySet.loadStatus.loading = false;
+      displaySet.loadStatus.loaded = true;
+    })
+    .catch(err => {
+      displaySet.loadStatus.loading = false;
+      throw err;
+    });
+
+  return promise;
+}
+
+async function _loadParametricMap({
+  extensionManager,
+  servicesManager,
+  displaySet,
+  headers,
+}: withAppTypes) {
+  const utilityModule = extensionManager.getModuleEntry(
+    '@ohif/extension-cornerstone.utilityModule.common'
+  );
+
+  const { dicomLoaderService } = utilityModule.exports;
```

Comment:
We can now import components from other extensions, eliminating the need for the utility module. Check if the `dicomLoaderService` is exported from the Cornerstone extension. If it is, import it at the top of the file. If not, export it from the Cornerstone extension.